src/Components/Search/Suggestions/SuggestionItemContainer.tsx

```diff
@@ -0,0 +1,12 @@
+import { Flex } from "@artsy/palette"
+import React, { SFC } from "react"
+
+interface Props {
+  children: JSX.Element[] | string[] | string
```

Comment:
I _think_ this can be `React.ReactNode`, which will cover all of these various cases.

Reply:
Yup, that works! Thank you.